packages/@jsii/benchmarks/lib/constants.ts

```diff
@@ -2,8 +2,8 @@ import * as path from 'node:path';
 
 export const fixturesDir = path.resolve(__dirname, '..', 'fixtures');
 
-export const cdkTagv2_21_1 = 'v2.21.1';
-export const cdkv2_21_1 = path.resolve(
+export const cdkTag = 'v2.31.0';
```

Comment:
❓ Question: Why removing the version post fix in the variable name? Or I guess: Any reason it was there in the first place?

I'm asking as this seems unrelated to the addressed issue.

Reply:
Great, that's what I though!